### DISCUSSION TITLE
Is there a way to zoom in the Tabs font?

### DISCUSSION THREAD

**Discussion by yuandu1990** [doublecmd/doublecmd#1566] [General]
May I know is there a way or configuration to zoom in Tabs font? I tried to increase the font size in all the items in "Font" configuration, but all don't work.
![image](https://github.com/doublecmd/doublecmd/assets/13115049/4832825c-4fbc-4bba-8164-2eb4475d88b0)

**Comment by alexx2000** [comment]
No, DC uses system font here.

**Reply by AdrianusWest** [reply]
![Screenshot_20240430_211226](https://github.com/doublecmd/doublecmd/assets/63318723/e908a4a4-50a5-4238-9fe0-d997d25abccb)
![Screenshot_20240430_211209](https://github.com/doublecmd/doublecmd/assets/63318723/af92aaf7-5c81-4a26-a7e4-76b22997af3c)


@alexx2000, I'm even afraid to wonder exactly what font is taken from the system for naming tabs, given that I have a maximum of 7-8 pins there and it looks marvelously shallow.
Arch Linux/KDE
extra/doublecmd-qt6 1.1.13-3

**Reply by Andrea-R-82** [reply]
It would be great to introduce the possibility of changing the font of the folder tabs and the Options menu. The one currently in use is quite small and reduces usability and readability.

**Reply by Laatmaarkomen** [reply]
Is there already a solution for the question ?

As mentioned the font of the Tabs is way to small. I tried to increase the windows font size, but that had no effect. I also tried to increase 'everything' in windows, but that was no succes.

The report above is dated January, I hope there is an update !

Thanks and greets from Holland.

**Reply by shishi** [reply]
I'm really looking forward to this setting too

**Reply by ags1234** [reply]
Me too.

**Comment by Andrea-R-82** [comment]
I think it would be better to decouple Doublecmd UI's font size from the system's one to allow a more granular control. This is particularly critical when switching between displays of different sizes. In these workflows, where you need to adapt the font size, it would be great to adjust it in real time with Ctrl + Mouse Wheel scroll, as you can do now with the path section above the tabs and the file panels.

That said, it's true that Doublecmd uses the system's font for these UI components, and the fact that they are all controlled by a single font size may be the root of the problem.
I no longer use Windows; now I use Linux Mint Cinnamon. In this OS, there are 5 types of fonts you can control with the Font Selection app: Default, Desktop, Document, Monospace, and Window title fonts. Changing the **Default** font changes the font size for both the folder tabs and the Options window in Double Commander (do it with DC closed).

<img width="1026" height="973" alt="doublecmd font size comparison" src="https://github.com/user-attachments/assets/bade0293-892b-4ea3-b902-c74060a29aa5" />

As you can see in the image above, on the right are Double Commander's window and its Options window; on the left, in the background for comparison, this Github page with its much smaller default font size.

**Comment by alexx2000** [comment]
Changing font personally for each element is overkill, I never seen a program which have so much font settings.

I suggest to add a "Default" font which will be used for all window elements (like Total Commander has), except file panels.
